registration-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializerCheckCompilerPluginComponent.scala

```diff
@@ -27,6 +28,7 @@ class SerializerCheckCompilerPluginComponent(
     s"Checks marked serializer for references to classes found in $serializerCheckPhaseName"
 
   private val serializerType = typeOf[Serializer]
+  private val serializerRegexFilter = typeOf[SerializerRegexFilter]
   private val serializabilityTraitType = typeOf[SerializabilityTrait]
```

Comment:
While you're here... I'd refine the error message a bit:

`not referenced in marked serializer` => `not registered` (?) `in a class annotated with @org.Serializer. This will lead to a missing codec for Akka serialization in the runtime`

Plus also maybe mention the regex? Since the class might be referenced but not within the desired regex